### PR TITLE
Productlist page added

### PR DESCRIPTION
**Pull request by sakshi006**
Product listing page added.
Responsive for all screen sizes.

Screenshot (548)
Screenshot (550)
Screenshot (551)

### CODE REVIEW

File: ProductListing/ProductListing.html [hunk 1]
```diff
@@ -37,6 +42,389 @@ <h2 class="logo"><a href="#">KairoShop</a></h2>
...
+                type="radio"
+                id="hightolow"
+                name="sortby"
+                value="High to Low"
+              />
+              <label for="hightolow">High To Low</label>
+            </div>
+          </form>
+        </div>
+        <div class="products-page">
```
Comment (sashaboi) [inline ProductListing/ProductListing.html:121]: In the PRD there seems to be some sort of a title which says Showing all products , maybe could be added ?
Also there seems to be a counter of current products displayed, that might be added as well.

File: index.html [hunk 2]
```diff
@@ -5,11 +5,18 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+
+    <style>
+        a{
+            margin: 0.5rem;
+        }
+    </style>
```
Comment (dusariudaykumar) [inline index.html:13]: I think it's better to use  external CSS

**Review by masraniJay — APPROVED** [review]
Looks good.

**Review by dusariudaykumar — CHANGES_REQUESTED** [review]
Looks good

File: index.html (hunk 2 above)
Comment (sakshi006) [reply index.html:13]: Hey, that is just temporary. I have to design the home page.

File: ProductListing/ProductListing.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+.clear {
+  text-decoration: underline;
+}
+
+.filter-head {
+  margin: 1rem 0rem;
+}
+
+.filter-category {
```
Comment (sashaboi) [inline ProductListing/ProductListing.css:27]: Maybe elements from filter category could have cursor pointer on hover? also maybe some sort of background color change one hover? could help UX I think

**Review by sashaboi — APPROVED** [review]
The overall page looks amazing great work ! 
I comment a few things which could be worked on when adding finishing touches, hope that helps, 
Onkar Deshpande

File: ProductListing/ProductListing.html (hunk 1 above)
Comment (sakshi006) [reply ProductListing/ProductListing.html:121]: Oh yes, forgot to add that. Thanks for pointing it out. :D

File: ProductListing/ProductListing.css (hunk 3 above)
Comment (sakshi006) [reply ProductListing/ProductListing.css:27]: Noted, will make the required changes.

**Pull request merged**